### ISSUE TITLE
LimitRange and ResourceQuota support for bandwidth resources

### ISSUE THREAD

**Issue by derekwaynecarr** [kubernetes/kubernetes#12562]
As a follow-on to 

https://github.com/GoogleCloudPlatform/kubernetes/pull/12187

We should be able to constrain bandwidth requests for a pod similar to other resources (memory, cpu, etc).  

Prerequisite to add request vs limit support in both LimitRange and ResourceQuota needs to happen first.

FYI @ncdc

**Comment by kargakis** [comment]
@kubernetes/sig-node-feature-requests

**Comment by vishh** [comment]
Bandwidth controlling has been deprecated and possibly removed from k8s already. This issue is no longer relevant.
